.github/workflows/coverity.yml

```diff
@@ -0,0 +1,40 @@
+name: Coverity
+on:
+  schedule:
+    - cron: '0 5 * * *' # Daily at 05:00 UTC
+
+jobs:
+  coverity:
+    name: "Test Suite"
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: osbuild
+    steps:
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+         path: osbuild
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get install python3-setuptools
+
+      - name: Download Coverity Tool
+        run: |
+          make coverity-download
+        env:
+         COVERITY_TOKEN: ${{ secrets.COVERITY_TOKEN }}
+         COVERITY_EMAIL: ${{ secrets.COVERITY_EMAIL }}
+
+      - name: Coverity check
+        run: |
+          make coverity-check
+
+      - name: Upload analysis results
+        run: |
+          make coverity-submit
```

Comment:
You sure this does not run the check twice? Doesn't `coverity-submit` depend on `COVERITY_ANALYSIS` and thus you could drop the `coverity-check` step?

Reply:
It looked ok:
```

Run make coverity-submit
fatal: No names found, cannot describe anything.
DESCRIPTION=
Submitting cov-analysis-osbuild.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  4 1321k    0     0    4 65535      0   272k  0:00:04 --:--:--  0:00:04  271k
100 1321k    0     0  100 1321k      0   856k  0:00:01  0:00:01 --:--:--  856k
100 1321k  100    30  100 1321k     18   818k  0:00:01  0:00:01 --:--:--  818k
Build successfully submitted.
```